src/flick/show/utils.py

```diff
@@ -65,11 +58,15 @@ def create_show_objects(show_info_lst):
                 show.cast = show_info.get("cast")
                 show.save()
                 if show_info.get("show_tags"):
-                    for tag_name in show_info.get("show_tags"):
+                    for genre in show_info.get("show_tags"):
```

Comment:
why was this changed to genre?

Reply:
genre now has the ext_api_id and the genre name so it is no longer tag_name